sections/sections-app/src/webapp/options.jsp

```diff
@@ -52,12 +61,24 @@
         </t:div>
     
         <t:div styleClass="act verticalPadding">
-            <h:commandButton
-                action="#{optionsBean.update}"
+            <h:commandButton                
                 value="#{msgs.update}"
                 styleClass="active"
                 rendered="#{optionsBean.sectionOptionsManagementEnabled}"
-                disabled="#{optionsBean.confirmMode}" />
+                disabled="#{optionsBean.confirmMode}" 
+                onclick="if(document.optionsForm[1].checked){
```

Comment:
using onclick is not ideal 👎

Reply:
What would be the correct way to do the same thing that the onclick does?

Reply:
you would move the logic to the Javascript file instead of embedding it within the onclick HTML attribute

Reply:
Updated